### PR TITLE
Adds support for Sigma Filters

### PR DESCRIPTION
**Pull request by sifex**
## Context

This PR adds the Sigma Filters feature to `sigma-cli` project. This relates to [the pySigma PR made here](https://github.com/SigmaHQ/pySigma/pull/226).

### Changelist:

- 
- A new section for pytest has been added to `pyproject.toml` to enforce pytest's relative working directory. This change is made to make working in tools like pyCharm easier – where the working directory is usually the directory where each test file is located, breaking relative file imports in CLI tests.

## Checklist

- [ ] `pyproject.toml` file updated with the pySigma 0.11.4 release (likely features release number).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -23,7 +23,8 @@ packages = [
 python = "^3.8"
 click = "^8.0.3"
 prettytable = "^3.1.1"
-pysigma = "^0.11.3"
+# pysigma = "^0.11.3""
+pysigma = { git = "https://github.com/SigmaHQ/pySigma", branch = "main" }
```
Comment (thomaspatzke) [inline pyproject.toml:27]: Switch back to the release and use pySigma 0.11.7 😉

Comment (sifex) [reply pyproject.toml:27]: Yea yea it's a draft PR 😂

File: sigma/cli/convert.py [hunk 2]
```diff
@@ -110,6 +108,12 @@ def fail(self, message: str, param, ctx):
     "-c",
     help="Select method for generation of correlation queries. If not given the default method of the backend is used."
 )
+@click.option(
```
Comment (thomaspatzke) [inline sigma/cli/convert.py:111]: Do we really need another option? Filters are "just" another type of input file that could be passed on the command line as argument.

Comment (sifex) [reply sigma/cli/convert.py:111]: Technically yes, if you go down it just appends the rules together but from a purely end-user experience, it makes sense to supply it as an argument. 

It comes from how I expect every input to map to some sort of output (rules, correlation rules) but technically filters don't produce an output.

**Pull request merged**